src/snapred/backend/recipe/algorithm/RawVanadiumCorrectionAlgorithm.py

```diff
@@ -124,6 +119,24 @@ def PyExec(self):
         self.chopNeutronData(self.inputVanadiumWS, self.outputVanadiumWS)
         self.chopNeutronData(self.inputBackgroundWS, self.outputBackgroundWS)
 
+        pcV = mtd[self.outputVanadiumWS].run().getProtonCharge()
+        pcB = mtd[self.outputBackgroundWS].run().getProtonCharge()
```

Comment:
Use `sef.mantidSnapper.mtd`

Using plain `mtd` should only be done in really simple algorithms without a `MantidSnapper` object